specification/hdinsight/resource-manager/Microsoft.HDInsight/preview/2025-01-15-preview/cluster.json

```diff
@@ -599,6 +608,9 @@
         "x-ms-examples": {
           "Enable HTTP connectivity": {
             "$ref": "./examples/HDI_Clusters_UpdateGatewaySettings_Enable.json"
+          },
+          "Enable update Entra User Information in an existing Entra User enabled HDInsight cluster": {
```

Comment:
Remove this "Enable", and pls also consider the relationship between description and generated sample name
I prefer to reduce this description to make it short.